shell.js

```diff
@@ -23,7 +23,7 @@ exports.pwd = common.wrap('pwd', _pwd);
 
 //@include ./src/ls
 var _ls = require('./src/ls');
-exports.ls = common.wrap('ls', _ls, {idx: 1});
+exports.ls = common.wrap('ls', _ls, {noGlob: true});
```

Comment:
We don't want to glob in common.wrap, we want to do it in late itself, because we want to optionally pass different opts to common.expand depending on the flags passed in.